jupyter_dashboards/nbextension/notebook/dashboard-view/dashboard.js

```diff
@@ -390,7 +389,30 @@ define([
         '</div>' +
         '<div class="grid-control-container grid-control-ne">' +
             '<i class="grid-control close-btn fa fa-close fa-fw"></i>' +
-            '<i class="grid-control add-btn fa fa-plus fa-fw"></i>' +
+            '<span class="grid-control add-btn add-btn-relative fa-fw" title="Add to the dashboard according to its position in the notebook">' +
```

Comment:
I think we can make the tooltip text a bit more concise: "Add to dashboard according to cell position in notebook"